Initium-Odp/src/com/universeprojects/miniup/server/GameUtils.java

```diff
@@ -884,8 +884,8 @@ public static String renderItem(ODPDBAccess db, HttpServletRequest request, Cach
 			if (strengthRequirement!=null && characterStrength<strengthRequirement)
 				hasRequiredStrength = false;
 			
-			double maxDura = (Double)item.getProperty("maxDurability");
-			double currentDura = (Double)item.getProperty("durability");
+			long maxDura = (Long)item.getProperty("maxDurability");
```

Comment:
It's possible for an item to have null dura, so long wont work. Needs to be Long.